webrtc.html

```diff
@@ -1689,6 +1718,12 @@ <h4>Set the RTCSessionDescription</h4>
                           </li>
                           <li>
                             <p>If <var>direction</var> is <code>"sendrecv"</code> or
+                            <code>"recvonly"</code>,
+                            set <var>transceiver</var>'s <a>[[\Receptive]]</a> slot
+                            to <code>true</code>, otherwise set it to <code>false</code>.</p>
```

Comment:
This is in the "remote description" steps, but don't we only want the slot set to `true` from *local* descriptions, as per [this comment thread](https://github.com/w3c/webrtc-pc/pull/1748#discussion_r165148079)?

Though I think it would still be correct to set the slot to `false` here.

Reply:
I agree, that's a mistake by me. I'll update.